modules/lazy-images/lazy-images.php

```diff
@@ -320,7 +322,76 @@ private static function get_placeholder_image() {
 		 */
 		return apply_filters(
 			'lazyload_images_placeholder_image',
-			'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7'
+			(
+				! empty( $image_dimensions )
```

Comment:
Instead of this nonsense, I suppose an easier approach might be to just initialize the `$image_dimensions` array so that both height and width are `1`. ¯\_(ツ)_/¯